ticdc/troubleshoot-ticdc.md

```diff
@@ -347,6 +351,26 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
 
-## How much PD storage does TiCDC use?
+## How can I find out whether a DDL statement fails to execute in downstream during TiCDC replication? How to resume the replication?
 
-TiCDC uses etcd in PD to store and regularly update the metadata. Because the time interval between the MVCC of etcd and PD's default compaction is one hour, the amount of PD storage that TiCDC uses is proportional to the amount of metadata versions generated within this hour. However, in v4.0.5, v4.0.6, and v4.0.7, TiCDC has a problem of frequent writing, so if there are 1000 tables created or scheduled in an hour, it then takes up all the etcd storage and returns the `etcdserver: mvcc: database space exceeded` error. You need to clean up the etcd storage after getting this error. See [etcd maintaince space-quota](https://etcd.io/docs/v3.4.0/op-guide/maintenance/#space-quota) for details. It is recommended to upgrade your cluster to v4.0.9 or later versions.
+Since v4.0.11, If a DDL statement fails to execute, the replication task (changefeed)automatically stops. The checkpoint-ts is the DDL statement's finish-ts minus one. If you want that TiCDC retries executing this statement in the downstream, use `cdc cli changefeed resume` to resume the replication task. For example:
```

Comment:
```suggestion
Since v4.0.11, If a DDL statement fails to execute, the replication task (changefeed) automatically stops. The checkpoint-ts is the DDL statement's finish-ts minus one. If you want TiCDC to retry executing this statement in the downstream, use `cdc cli changefeed resume` to resume the replication task. For example:
```